src/components/include/sl_thd.h

```diff
@@ -17,20 +17,49 @@ typedef enum {
 	SL_THD_DYING,
 } sl_thd_state;
 
+typedef enum {
+	SL_FLAG_OWNTC = 1,      /* Thread owns a tcap */
```

Comment:
Kk, I don't really have better ideas for naming. I'll go with `SL_THD_PROPERTY_XXXX`, it sounds good. Will fix the enum typedef naming too.